apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -275,6 +275,18 @@ public final class AppConfiguration extends BaseAppConfiguration {
    */
   public static final String SSM_PATH_PROCESS_DLQ = "rda/job/process_dlq";
 
+  /**
+   * The path of the SSM parameter that should be used to determine if the cleanup tasks that remove
+   * old FISS and MCS claims should be run on subsequent job runs.
+   */
+  public static final String SSM_PATH_RUN_CLEANUP = "rda/job/cleanup/run_cleanup";
+
```

Comment:
Very minor nit.  `enabled` might be more obvious in its meaning than `run_cleanup` especially since we also have `run_size`.

Reply:
Done